Framework/Configuration/Settings.cs

```diff
@@ -13,7 +13,7 @@ public static class Settings
         public static readonly string ClientSeed = Conf.GetString("ClientSeed", "179D3DC3235629D07113A9B3867F97A7");
         public static readonly ClientVersionBuild ClientBuild = Conf.GetEnum("ClientBuild", ClientVersionBuild.V2_5_2_40892);
         public static readonly ClientVersionBuild ServerBuild = Conf.GetEnum("ServerBuild", ClientVersionBuild.V2_4_3_8606);
-        public static readonly string ServerAddress = Dns.GetHostAddresses(Conf.GetString("ServerAddress", "127.0.0.1")).GetValue(0).ToString();
+        public static readonly string ServerAddress = Dns.GetHostAddresses(Conf.GetString("ServerAddress", "127.0.0.1")).First().ToString();
```

Comment:
Not sure about this, this can be null and throw an exception.

Reply:
Yeah, I guess I am just not sure about the entire Dns resolving :D Keep it like this 👍